### PR TITLE
Load DVTServices and DVTPortal frameworks

### PR DESCRIPTION
**Pull request by neonichu**
IDEFoundation depends on them in Xcode 7.3

cc @segiddins

### CODE REVIEW

**Comment by neonichu** [comment]
Since errors in all but the last `dlopen` are ignored, this also works with previous versions of Xcode which did not have those frameworks.

**Comment by manuyavuz** [comment]
I've verified that this fix is working on my project which was having problem of converting project to XML format on some cases. :+1: :ship:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Xcodeproj Changelog
 
+## Master
+
+##### Bug Fixes
+
+* Fix DevToolsCore based serialization on Xcode 7.3.
```
Comment (segiddins) [inline CHANGELOG.md]: `Fix ASCII .xcodeproj serialization with Xcode 7.3`

**Comment by segiddins** [comment]
👍🏻

**Comment by marcelofabri** [comment]
Any chances to cut a `0.28.3` release with this fix? :grimacing:

**Pull request merged**